index.html

```diff
@@ -1,224 +1,299 @@
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <ul class="skills-details">
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="project-details" type="button">See project</button>
+        </div>
+      </div>
+      <div class="cards">
+        <div class="card-item">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <ul class="skills-details">
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="project-details" type="button">See project</button>
+        </div>
+      </div>
+      <div class="cards">
+        <div class="card-item">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <ul class="skills-details">
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="project-details" type="button">See project</button>
+        </div>
+      </div>
     </section>
     <!-- ----------------------About section----------------->
-    <section class="about-section">
-        <div class="about-introduction">
-            <div class="about-introduction-content">
-                <h1 class="about-heading">About me</h1>
-                <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some
-                    of my work and experience! If you like what you see and have a project you need coded, don't
-                    hesitate to contact me.
-                </p>
-                <a href="https://www.linkedin.com/in/kifle-haile-5a613761/" target="_blank" rel="noopener noreferrer">
-                    <button class="my-resume-button" id="resume-button" type="button">Get My Resume</button>
-                </a>
-            </div>
-            <div class="background-graphic"></div>
+    <section id="about-section" class="about-section">
+      <div class="about-introduction">
+        <div class="about-introduction-content">
+          <h1 class="about-heading">About me</h1>
+          <p>
+            Hello I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don't
+            hesitate to contact me.
+          </p>
+          <a
+            href="https://www.linkedin.com/in/kifle-haile-5a613761/"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
```

Comment:
- [ ] Kindly make sure to add the `:hover` and `:active` state to this button as shown in the Figma design: 🎯
      ![image](https://user-images.githubusercontent.com/104968236/229135249-182155c5-788a-4b69-ac93-6e38e9eeaf2f.png)

Reply:
Thank you so much for all your comments. It allows me to improve myself and the quality of my work.